client/src/pages/InsightsPage/InsightsPage.js

```diff
@@ -0,0 +1,111 @@
+import React, { useState } from "react";
+import styles from "./InsightsPage.module.css";
+import { DateSelector } from "../../components/DateSelector/DateSelector.js";
+import { TaskDurationBarChart } from "../../components/TaskDurationBarChart/TaskDurationBarChart.js";
+
+// helpers for sample -- to be removed later
+const nHoursAhead = (nHours) => {
+  const nextHour = new Date();
+  nextHour.setTime(nextHour.getTime() + nHours * 60 * 60 * 1000);
+  return nextHour;
+};
+
+// this simulates an API call response
+const sampleTaskData = [
+  {
+    title: "Task 1",
+    description: "Task 1 description",
+    startDate: new Date(),
+    endDate: new Date(),
+    originalStartDate: new Date(),
+    originalEndDate: new Date(),
+    taskStatus: "NOT STARTED", // only tasks that are done will shown in the bar graph
+  },
+  {
+    title: "Task 2",
+    description: "Task 2 description",
+    startDate: new Date(),
+    endDate: nHoursAhead(3),
+    originalStartDate: new Date(),
+    originalEndDate: nHoursAhead(2),
+    taskStatus: "DONE",
+  },
+  {
+    title: "Task 3",
+    description: "Task 3 description",
+    startDate: new Date(),
+    endDate: nHoursAhead(1),
+    originalStartDate: new Date(),
+    originalEndDate: nHoursAhead(3),
+    taskStatus: "DONE",
+  },
+  {
+    title: "Task 2",
+    description: "Task 2 description",
+    startDate: new Date(),
+    endDate: nHoursAhead(3),
+    originalStartDate: new Date(),
+    originalEndDate: nHoursAhead(2),
+    taskStatus: "DONE",
+  },
+  {
+    title: "Task 3",
+    description: "Task 3 description",
+    startDate: new Date(),
+    endDate: nHoursAhead(1),
+    originalStartDate: new Date(),
+    originalEndDate: nHoursAhead(3),
+    taskStatus: "DONE",
+  },
+  {
+    title: "Task 2",
+    description: "Task 2 description",
+    startDate: new Date(),
+    endDate: nHoursAhead(3),
+    originalStartDate: new Date(),
+    originalEndDate: nHoursAhead(2),
+    taskStatus: "DONE",
+  },
+  {
+    title: "Task 3",
+    description: "Task 3 description",
+    startDate: new Date(),
+    endDate: nHoursAhead(1),
+    originalStartDate: new Date(),
+    originalEndDate: nHoursAhead(3),
+    taskStatus: "DONE",
+  },
+];
+
+const InsightsPage = () => {
+  const [currDate1, setCurrDate1] = useState(new Date());
```

Comment:
I would change `currDate1` and `currDate2` to something that's more appropriate. Perhaps something like startDate/endDate might work? (Depends on the context of what these dates are being used for)

Reply:
These are just dummy dates for the purposes of showing the two types of `DateSelector`. This will not be part of the final `InsightsPage`.

Reply:
Noted